test/parallel/test-fs-long-path.js

```diff
@@ -1,4 +1,7 @@
 'use strict';
+
+if (process.platform !== 'win32') process.exit();
```

Comment:
Can you use the TAP-compatible approach we use e.g. [here](https://github.com/nodejs/node/blob/v5.1.0/test/parallel/test-spawn-cmd-named-pipe.js#L6-L9)?

(I forgot about `common.isWindows`, it's fairly new.)